exercice/exercice-one.js

```diff
@@ -0,0 +1,41 @@
+// 1. Write a function(s) that checks the level of nesting of an array. Then, use that function to flatten the array into a single-dimensional array.
+
+function getNestingLevel(array) {
+  if (!Array.isArray(array)) {
+    return 0;
+  }
```

Comment:
- [ ] kindly use arry return by making your code clean and short 😃 hint: how about thinking about making your iff on a single line 🤔

Reply:
yes on it